PWGHF/Core/HfMlResponseDsToKKPi.h

```diff
@@ -0,0 +1,198 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file HfMlResponseDsToKKPi.h
+/// \brief Class to compute the ML response for Ds∓ → K∓ K∓ π± analysis selections
+/// \author Samuele Cattaruzzi <samuele.cattaruzzi@cern.ch>
+
+#ifndef PWGHF_CORE_HFMLRESPONSEDSTOKKPI_H_
+#define PWGHF_CORE_HFMLRESPONSEDSTOKKPI_H_
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "CommonConstants/PhysicsConstants.h"
```

Comment:
I left them because I saw they are use also in the HfMLResponse of the D0 and D+ and I thought they are necessary for the FILL_MAP_DS(FEATURE). However I have removed them and the compilation succeeded

Reply:
In general, you should always explicitly include headers for everything you explicitly need and nothing else.

Reply:
You still include `PhysicsConstants.h`. Where do you use it?